src/web-auth/index.js

```diff
@@ -387,7 +388,7 @@ WebAuth.prototype.checkSession = function(options, cb) {
     .with(options);
 
   if (params.responseType === 'code') {
-    return cb(new Error("responseType can't be `code`"));
+    return cb({ error: 'error', error_description: "responseType can't be `code`" });
```

Comment:
Why has this changed?

Reply:
it was wrong before. you couldn't catch the error because it was not using the standard error formatting `{error, error_description}`

Reply:
ok good was wondering that